biblib/tests/base.py

```diff
@@ -314,16 +319,17 @@ def setUp(self):
 
         current_app.logger.info('Setting up db on: {0}'
                                 .format(current_app.config['SQLALCHEMY_BINDS']))
-        db.create_all()
+        Base.metadata.create_all(bind=self.app.db.engine)
 
     def tearDown(self):
         """
         Remove/delete the database and the relevant connections
 
         :return: no return
         """
-        db.session.remove()
-        db.drop_all()
+        self.app.db.session.remove()
+        Base.metadata.drop_all(bind=self.app.db.engine)
+        #self.app.db.drop_all()
```

Comment:
There are a couple commented-out code bits leftover from testing that should be removed. (see also https://github.com/adsabs/biblib-service/pull/113/files#diff-2e1a024c02ade1c733a8195d5f68fa61R16)